### ISSUE TITLE
Question about classifier_free_guidance.py

### ISSUE THREAD

**Issue by TreeberryTomato** [lucidrains/denoising-diffusion-pytorch#215]
May I ask why in [classifier_free_guidance.py](https://github.com/lucidrains/denoising-diffusion-pytorch/blob/main/denoising_diffusion_pytorch/classifier_free_guidance.py) you let `null_classes_emb=nn.Parameter(torch.randn(dim))` instead of `torch.zeros(dim)`, because the latter one seems more reasonable.
